examples/tests/test_examples.py

```diff
@@ -49,13 +47,7 @@ def emit(self, record):
 
 
 # Initialize the device, to avoid Rust warnings from showing in the first example
-gfx.renderers.wgpu.get_shared()
-
-
-def test_that_we_are_on_lavapipe():
-    print(wgpu_backend)
-    if os.getenv("PYGFX_EXPECT_LAVAPIPE"):
-        assert is_lavapipe
```

Comment:
I see. I can adjust if you want to revisit this. But I feel like the font variations might e a reason to avoid this strategy.

Perhaps we can get lavapipe built and installed from conda-forge. But that seems to be a whole other project.

Your `Screenshots / Regenerate` CI is a good bandaid for now.

Reply:
It seems unreasonably complicated to exactly replicate results on various systems, there are many more variables to control than only lavapipe.

Instead we'll introduce an image similarity metric that is insensitive to the kind of errors we want to ignore, I'm working on that in the background. It's not a high priority.